### PR TITLE
cfitsio400 with patches

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR is a copy of #13 with extra patches to fix the build.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #18

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
I have no idea how this plays with the other open PRs for this feedstock, happy to close or contribute to other PRs as appropriate.

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml:33]: Pls use this for openmp: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=openmp#openmp

**Review by beckermr — CHANGES_REQUESTED** [review]
We can rebase this PR on top of the other ones once they are merged.

File: recipe/meta.yaml
Comment (duncanmmacleod) [reply recipe/meta.yaml]: This has been addressed in the latest force push, thanks @beckermr.

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Review by beckermr — APPROVED** [review]

**Comment by beckermr** [comment]
@duncanmmacleod Anything else you want to do here?

**Comment by duncanmmacleod** [comment]
> @duncanmmacleod Anything else you want to do here?

Nope, I'm just here to hurry the cfitsio migration along. Happy to merge.

**Pull request merged**
